packages/react-scripts/scripts/utils/forkSsr.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * This script runs a Webpack compiler to watch the SSR build. It is designed to
+ * be run by child_process.fork from the start-ssr script.
+ */
+
+const fs = require('fs');
+const webpack = require('webpack');
+const { prepareUrls } = require('react-dev-utils/WebpackDevServerUtils');
+const paths = require('../../config/paths');
+
+const ssrConfigFactory = require('../../config/webpack.config.ssr');
+const statusFile = require('./statusFile');
+const { DebugCompilerUi, ProcessSendCompilerUi } = require('./MultiCompilerUi');
+const { createCustomCompiler } = require('./customWebpackUtils');
+
+const useYarn = fs.existsSync(paths.yarnLockFile);
+
+const config = ssrConfigFactory('development');
+const appName = require(paths.appPackageJson).name;
+
+const useTypeScript = fs.existsSync(paths.appTsConfig);
+const tscCompileOnError = process.env.TSC_COMPILE_ON_ERROR === 'true';
+
+// We don't start a DevServer so we can use dummy URLs
+const urls = prepareUrls(
+  'http', // protocol
+  'localhost', // HOST
+  '3000', // port
+  paths.publicUrlOrPath.slice(0, -1)
+);
+
+// We don't start a DevServer so we can drop devSocket messages
+const devSocket = {
+  warnings: () => {},
+  errors: () => {},
+};
+
+const ui = process.send ? new ProcessSendCompilerUi() : new DebugCompilerUi();
+
+const compiler = createCustomCompiler(ui, {
+  appName,
+  config,
+  devSocket,
```

Comment:
In response to the above this can simply be 

```suggestion
  devSocket: undefined,
```

Reply:
This would fail if we one day enabled Typescript though right? The `devSocket` is used to send Typescript compiler errors.

Reply:
That is true, I guess when this was originally defined in the start-ssr it was based on our currently used and standard langs in Skyscanner and prod standards.